spring-integration-mongodb/src/main/java/org/springframework/integration/mongodb/store/AbstractConfigurableMongoDbMessageStore.java

```diff
@@ -107,6 +110,15 @@ public AbstractConfigurableMongoDbMessageStore(MongoDatabaseFactory mongoDbFacto
 		this.mappingMongoConverter = mappingMongoConverter;
 	}
 
+	/**
+	 * Define the option to auto create indexes or not.
+	 * @param createIndex a boolean.
+	 * @since 6.0.8.
```

Comment:
I am wondering if this is the good version. The issue was found in the `version 6.1.1`.

Reply:
The issue is still apply for an older version as well.
Since it is in support, we still have to back port it.
Therefore we need to use as lower version for `@since` as we back port the fix.